app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -406,7 +406,7 @@ class _TextInputWidgetConsumerState extends ConsumerState<_TextInputWidget> {
       style: Theme.of(context).textTheme.bodySmall,
       cursorColor: Theme.of(context).colorScheme.tertiary,
       maxLines:
-          MediaQuery.of(context).orientation == Orientation.portrait ? 6 : 2,
+          MediaQuery.of(context).orientation == Orientation.portrait ? 6 : 6,
```

Comment:
if something then 6 or otherwise 6 .... ? I know a shortcut:

```suggestion
          6,
```